src/weights.jl

```diff
@@ -291,3 +291,81 @@ end
 
 wmedian(v::RealVector, w::RealVector) = median(v, weights(w))
 wmedian{W<:Real}(v::RealVector, w::WeightVec{W}) = median(v, w)
+
+
+
+
+###### Weighted quantile #####
+# type 7 for now, as in base
```

Comment:
Could you move this mention of "type 7" to the docs? Ideally, if you could give a reference about this (and alternative) definitions, that would be very helpful.